models/prices/prices_tokens.sql

```diff
@@ -1789,6 +1789,7 @@ VALUES
     ("luna-luna-wormhole", "polygon", "LUNA", "0x9cd6746665d9557e1b9a775819625711d0693439", 6),
     ("ust-terrausd", "polygon", "USTC", "0x692597b009d13c4049a947cab2239b7d6517875f", 18),
     ("maticx-liquid-staking-matic-pos","polygon","MATICX","0xfa68fb4628dff1028cfec22b4162fccd0d45efb6",18),
+    ("gns-polygon", "polygon", "GNS", "0xe5417af564e4bfda1c483642db72007871397896", 18),
```

Comment:
It seems `gns-polygon` is currently not available on coinpaprika api, or wrong token-id.
https://github.com/duneanalytics/spellbook/actions/runs/4310407488/jobs/7518804178#step:6:30

Reply:
I will replace the token-id using `gns-gains-network`.
https://api.coinpaprika.com/v1/coins/gns-gains-network